src/syntax/parser.rs

```diff
@@ -222,13 +222,13 @@ where
     Ok(attributes)
 }
 
-fn get_identifier<I>(ps: &mut ParserStream<I>) -> Result<ast::Identifier>
+fn get_identifier<I>(ps: &mut ParserStream<I>, allow_private: bool) -> Result<ast::Identifier>
```

Comment:
For readability, I would very much like to see two convenience wrappers around `get_identifier`: `get_public_identifier` and `get_private_identifier`.